app/src/unitTest/kotlin/batect/docker/DockerAPISpec.kt

```diff
@@ -121,7 +121,8 @@ object DockerAPISpec : Spek({
                 val image = DockerImage("the-image")
                 val network = DockerNetwork("the-network")
                 val command = listOf("doStuff")
-                val request = DockerContainerCreationRequest(image, network, command, "some-host", "some-host", emptyMap(), "/some-dir", emptySet(), emptySet(), HealthCheckConfig(), null, false, false, emptySet(), emptySet())
+                val entrypoint = listOf("sh")
+                val request = DockerContainerCreationRequest(image, network, command, entrypoint, "some-host", "some-host", emptyMap(), "/some-dir", emptySet(), emptySet(), HealthCheckConfig(), null, false, false, emptySet(), emptySet())
```

Comment:
Not part of this PR, but I need to come back and do something about this, that parameter list is getting way out of hand...